public/js/main.js

```diff
@@ -15,6 +16,7 @@ import highlight from './highlight';
 if (document.readyState === 'interactive' || document.readyState === 'complete') {
```

Comment:
None of the custom JS for the Registry is built to autoinit, so I don't think it would add anything to use autoinit here instead of what's being done...